rl4co/models/zoo/mdam/encoder.py

```diff
@@ -63,6 +65,7 @@ def forward(self, x, mask=None, return_transform_loss=False):
 
         h_embeded = x
         h_old = self.layers(h_embeded)
+        # ValueError: too many values to unpack (expected 3); here self.attention_layer(h_old) is a single tensor
         h_new, attn, V = self.attention_layer(h_old)
```

Comment:
Since it was working, was something changed here?

Reply:
I guess the behavior of `MultiHeadAttention` was changed, was it? @cbhua 
Because in this model they remade the MHA class from what I remember